python/dataset.cpp

```diff
@@ -702,6 +702,16 @@ void init_dataset(py::module &m) {
         :param x: Realigned data to histogram.
         :return: Histogramed data.
         :rtype: Dataset)");
+  m.def("map", event::map, py::arg("function"), py::arg("iterable"),
+        py::call_guard<py::gil_scoped_release>(),
+        R"(Return mapped event data.
+
+        This only supports event data.
+
+        :param function: Data array serving as a descretized mapping function.
```

Comment:
`descritized` -> `discretized`

Reply:
That is already what's there?

Reply:
Not what I'm seeing. Still `descretized` in latest.

Reply:
oops, you are right, was looking at the wrong "i".